d2l-table-style.html

```diff
@@ -286,7 +286,6 @@
 				position: sticky;
 				top: -5px;
 				border-left: none;
-				border-top: none;
```

Comment:
this fixes problem 3 (see main description of PR)

Reply:
This causes double thick borders on multi-row headers. 

![image](https://user-images.githubusercontent.com/15876659/37472332-2dc1a8e6-2842-11e8-91d3-5784ebe94e9c.png)